assets/js/googlesitekit/notifications/components/common/ActionsCTALinkDismiss.js

```diff
@@ -35,34 +43,51 @@ export default function ActionsCTALinkDismiss( {
 	onCTAClick,
 	onDismiss = () => {},
 	dismissLabel = __( 'OK, Got it!', 'google-site-kit' ),
+	dismissOnCTAClick = true,
 	dismissExpires = 0,
 	dismissOptions = {},
+	showCTAError = true,
 } ) {
 	const isNavigatingToCTALink = useSelect( ( select ) => {
 		return ctaLink
 			? select( CORE_LOCATION ).isNavigatingTo( ctaLink )
 			: false;
 	} );
 
+	const ctaError = useSelect( ( select ) => {
+		return select( CORE_SITE ).getError( 'notificationCTAClick', [ id ] );
+	} );
+
 	return (
-		<div className={ className }>
-			<CTALink
-				id={ id }
-				ctaLink={ ctaLink }
-				ctaLabel={ ctaLabel }
-				onCTAClick={ onCTAClick }
-				dismissExpires={ dismissExpires }
-			/>
+		<Fragment>
+			{ showCTAError && ctaError && (
+				<ErrorText message={ ctaError.message } />
+			) }
+			<div className={ className }>
+				<CTALink
+					id={ id }
+					ctaLink={ ctaLink }
+					ctaLabel={ ctaLabel }
+					onCTAClick={ onCTAClick }
+					dismissOnCTAClick={ dismissOnCTAClick }
+					dismissExpires={ dismissExpires }
+					showError={ false }
```

Comment:
The `CTALink` component will set the error, but we don't want it to be displayed by that component as we'll display it in this one instead, above both the CTAs (primary & dismiss).

Reply:
From what I can see currently, we are rendering the error in both, the `ActionsCTALinkDismiss` and `CTALink` components.

Reply:
Sorry @jimmymadon, I wasn't clear enough in my initial comment - I was trying to point out that due to passing `showError={false}` to `CTALink` the error won't be displayed in that component, whereas it will in this one (unless this component's `showCTAError` prop is truthy).